src/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -14,6 +14,8 @@ internal static class Constants
 
         public const string UapWEBRedirectUri = "https://sso"; // only ADAL supports WEB
         public const string DefaultRedirectUri = "urn:ietf:wg:oauth:2.0:oob";
+        public const string DefaultDesktopRedirectUri = "https://login.microsoftonline.com/common/oauth2/nativeclient";
+        public const string DefaultNetCoreRedirectUri = "https://localhost";
```

Comment:
>DefaultNetCoreRedirectUri [](start = 28, length = 25)

Why this? Why can't this be the same as the one above?

Reply:
not sure but it is according to the spec outlined here
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1186

Reply:
Perhaps we need to have a WithLocalRedirectUri

---
In reply to: [297898748](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1231#discussion_r297898748) [](ancestors = 297898748)

Reply:
@JM ... not sure we should do this. 

---
In reply to: [297899005](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1231#discussion_r297899005) [](ancestors = 297899005)

Reply:
Will be address in a later PR